### PR TITLE
update: new CI/CD workflows for deployment

### PR DESCRIPTION
**Pull request by Tremirre**
Plz approve

### CODE REVIEW

**Comment by Tremirre** [comment]
Testing if conversation resolution is working

**Review by zuzg — APPROVED** [review]
╰(*°▽°*)╯

File: .github/workflows/prod-deploy.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+name: Deploy to Production
+
+on:
+  push:
```
Comment (Tremirre) [inline .github/workflows/prod-deploy.yml:4]: Testing if conversation resolution is working v2

Comment (Tremirre) [reply .github/workflows/prod-deploy.yml:4]: It is working fine :3

**Review by azywot — APPROVED** [review]
LGTM, GJ Bartosh ٩(-̮̮̃•̃)۶

**Pull request merged**